### PR TITLE
CONTRIBUTING.md: remove integration-tests, include spread

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
+We need to verify that the code functionality and quality is not degraded by additions before merging any changes to snapd's codebase. For each PR we run checks in three different groups: static, unit and spread.
 
-Depending on the affiliation of the GitHub user submitting a PR, the following actions may happen after receiving it:
+Static test use several code analysis tools present in the GoLang ecosystem (go vet, go lint and go fmt) to make sure that the code always aligns with the standards. They also check the markdown format of documentation files. All the existing unit tests are also executed, and the coverage info is reported to coveralls. Regarding [spread](https://github.com/snapcore/spread), we use it to verify the integrity of the product exercising it as a whole, both from an end user standpoint (eg. all kind of interactions with the snap tool from the command line) and from a more systemic approach (testing upgrades, for instance).
```
Comment (chipaca) [inline CONTRIBUTING.md]: can we wrap these at ~70, please? makes reading the raw text a lot easier, and doesn't affect the rendered output

**Comment by chipaca** [comment]
:+1: (even more so if wrapped ;-) )

**Comment by fgimenez** [comment]
@chipaca done, thx!

**Comment by chipaca** [comment]
Oooh! While I'm being picky, you could *also* change the long

```
Before contributing you should sign [Canonical's contributor agreement](http://www.ubuntu.com/legal/contributors),
it’s the easiest way for you to give us permission to use your contributions.
```

to

```
Before contributing you should sign [Canonical's contributor agreement][1],
it’s the easiest way for you to give us permission to use your contributions.
```

and then have a footnote-like
```
[1]: http://www.ubuntu.com/legal/contributors
```

:-D

**Comment by chipaca** [comment]
(you can also use text, e.g. `CLA`, instead of a number)

**Comment by fgimenez** [comment]
@chipaca sure! pls take another look, thx :)

**Comment by zyga** [comment]
Looks nice, thank you for improving this

**Comment by chipaca** [comment]
@fgimenez I hate to say this, but you broke it :-(

Note that your `[1]` reference appears as is in the rendered markdown. If instead you had left the brackets before, i.e. `[Canonical's contributor agreement][1]` the rendered output would have an actual link, which is what we want, and the raw output would be readable, which is a nice-to-have.

**Comment by fgimenez** [comment]
@chipaca sorry, didn't get your comment right, should be fixed now :)

**Pull request merged**